File: plugins/de.ovgu.featureide.examples/featureide_examples/GPL-FH-Java/features/StronglyConnected/GPL/Vertex.java
package GPL;

import java.util.LinkedList;
import java.util.Collections;
import java.util.Comparator;

// of Graph

  // ***********************************************************************
   
public class Vertex {
    public int finishTime;
    public int strongComponentNumber;
      
    public void display() {
        System.out.print( " FinishTime -> " + finishTime + " SCCNo -> " 
                        + strongComponentNumber );
        original();
    }
}

File: plugins/de.ovgu.featureide.examples/featureide_examples/GPL-FH-CSharp/features/MSTKruskal/Vertex.cs
//layer MSTKruskal;

//import java.lang.Integer;
using System.Collections;
//import java.util.Collections;
//import java.util.Comparator;

// of Graph

  // *************************************************************************

public class Vertex {//refines
    public  Vertex representative;
    public ArrayList members;

    public void display() {
        if ( representative == null )
            System.Console.Out.Write( "Rep null " );
        else
            System.Console.Out.Write( " Rep " + representative.GetName() + " " );
        original();
    }
}
